op-chain-ops/cmd/op-upgrade/main.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"slices"
+
+	"golang.org/x/exp/maps"
+
+	//"strings"
```

Comment:
dead code

Reply:
this comment is blocking mergify auto-merging fyi - not sure if this is intentional